pymc3/tests/test_sampling.py

```diff
@@ -693,6 +694,16 @@ def test_exec_nuts_init(method):
         assert "a" in start[0] and "b_log__" in start[0]
 
 
+@pytest.fixture(scope="class")
+def point_list_arg_bug_fixture() -> Tuple[pm.Model, pm.backends.base.MultiTrace]:
+    with pm.Model() as pmodel:
+        n = pm.Normal('n')
+        trace = pm.sample()
```

Comment:
I'm new to fixtures, but since this test doesn't need convergence, we can save test time with `pm.sample(tune=100, draws=50, cores=1, chains=1)`